### ISSUE TITLE
Animated Camera's binding time - need cascade lock

### ISSUE THREAD

**Issue by npolys** [x3dom/x3dom#915]
When a user binds to a Viewpoint that is under an actively-animated transform,
 the timestep / event cascade is not locked... so the user gets a view with some spatial offset 

Examples here: 
note X3DOM PG Up-PG Dwn  like when you jump on the helicopters

http://metagrid2.sv.vt.edu/~keren9/Chopper/pages/directional_tablePage.html

**Comment by tsturm** [comment]
@npolys Can you try it again with the current dev version?

https://x3dom.org/download/dev/

**Comment by andreasplesch** [comment]
The animated viewpoints work fine with an instant transition of transitionType='"TELEPORT"'. This is confirming the analysis, The target view is determined at the time when the transition starts but once it ends the target has moved away. But I think there is some additional issue, let's see.

**Comment by andreasplesch** [comment]
No, it looks like properly updating the target for the transition with #1092 fixes the issue. @npolys, please try the latest dev when you have a chance.
